rules.mk

```diff
@@ -55,9 +55,9 @@ build/defaultDb.h: $(wildcard golpe.yaml src/apps/*/golpe.yaml)
 	PERL5LIB=golpe/vendor/ golpe/external/rasgueadb/rasgueadb-generate golpe.yaml build
 
 clean:
-	rm -f $(BIN) src/*.o src/*.d src/apps/*/*.o src/apps/*/*.d
+	rm -f $(BIN) src/*.{o,d} src/apps/*/*.{o,d}
 	rm -rf build/
-	rm -f golpe/external/uWebSockets/src/{*.o,libuWS.a,libuWS.so}
```

Comment:
`libuWS.a` and `libuWS.so` didn't reside in `uWebSockets/src`, but in [`uWebSockets/src`](https://github.com/paplorinc/golpe/blob/master/rules.mk#L10)